logbook-api/src/main/java/org/zalando/logbook/attributes/AttributeExtractor.java

```diff
@@ -0,0 +1,26 @@
+package org.zalando.logbook.attributes;
+
+import org.apiguardian.api.API;
+import org.zalando.logbook.HttpRequest;
+import org.zalando.logbook.HttpResponse;
+
+import javax.annotation.Nonnull;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+@API(status = EXPERIMENTAL)
+public interface AttributeExtractor {
+
+    @SuppressWarnings("RedundantThrows")
```

Comment:
Sorry, one more question: do we really want to have a checked exception in the interface? I understand that JWT related extractors need to deal with checked exceptions, but incorporating Exception in an interface signature seems to be unnecessary. Otherwise, most Logbook interfaces would have it in their method signatures.

Reply:
Your comments are really useful and always welcome, please don't apologize 😊
I'll fix it ASAP.

Reply:
Done: 3ebef9f.